src/modules/fundingManager/bondingCurve/abstracts/RedeemingBondingCurveBase_v1.sol

```diff
@@ -269,14 +269,24 @@ abstract contract RedeemingBondingCurveBase_v1 is
         if (collateralRedeemAmount < _minAmountOut) {
             revert Module__BondingCurveBase__InsufficientOutputAmount();
         }
-        // Transfer tokens to receiver
-        collateralToken.safeTransfer(_receiver, collateralRedeemAmount);
+
+        // Use virtual function to handle collateral tokens
+        _handleCollateralTokensAfterSell(_receiver, collateralRedeemAmount);
+
         // Emit event
         emit TokensSold(
             _receiver, _depositAmount, collateralRedeemAmount, _msgSender()
         );
     }
 
+    /// @notice Virtual function to handle collateral tokens after a successful sell.
+    /// @param  _receiver The address that used the sell function.
```

Comment:
It's just the address receiving the tokens, not necessarily the address that called the sell function (`SellTo`).

Reply:
See comment above